compiler/rustc_resolve/src/lib.rs

```diff
@@ -64,6 +65,7 @@ use macros::{MacroRulesBinding, MacroRulesScope, MacroRulesScopeRef};
 
 use crate::effective_visibilities::EffectiveVisibilitiesVisitor;
 
+type ImportId<'a> = Interned<'a, NameBinding<'a>>;
```

Comment:
This is not an import ID in general, it's an import ID in effective visibility tables specifically.
I'd keep it in its old place.